source/standards/dns-hosting.html.md.erb

```diff
@@ -31,3 +31,18 @@ You can read more about [service domains and DNS][] in the Service Manual.
 [Amazon Route 53]: https://aws.amazon.com/route53/
 [Google Cloud DNS]: https://cloud.google.com/dns/
 [service domains and DNS]: https://www.gov.uk/service-manual/technology/get-a-domain-name
+
+## Production and non-production domain names
+
+Your non-production domains should be subdomains of your production domain.
+
+For example, if you service can be found at `myservice.service.gov.uk`, then you might also have Route 53 records for your other environments, such as:
```

Comment:
This point's under discussion at the moment. I agree with it, but I don't think it's been properly agreed

Reply:
Happy to leave this open assuming the discussion isn't gonna take forever, LMK :)